pallets/credit-accumulation/Cargo.toml

```diff
@@ -23,6 +23,7 @@ node-primitives = {version = '2.0.0', default-features = false, path = "../../pr
 pallet-balances = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", default-features = false}
 pallet-credit = { default-features = false, path = "../credit", version = "3.0.0" }
 pallet-deeper-node = { version = "3.0.0", default-features = false, path = "../deeper-node" }
+pallet-uniques = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", default-features = false}
```

Comment:
I found  that `pallet-uniques` was added to `credit-accumulation`, `micropayment`, `staking`, `tips` and I think these crates don't need to change.
You are using tightly coupling, maybe using loosely coupling will solve this problem
https://docs.substrate.io/v3/runtime/pallet-coupling/